pkg/kudoctl/cmd/diagnostics/testdata/cluster_role.yaml

```diff
@@ -1,23 +1,4 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
```

Comment:
on the one hand, these test files were pretty verbose. on the other, these were real cluster resources and the tests were as close to the real thing as possible

Reply:
But this makes it hard to update the test files if new functionality is added to diagnostics. And because the tests only check for the existence of files based on information gathered through _some_ fields of these resources (e.g. by filtering with the `kudo.dev/operator` label), it's much clearer to _only_ provide these fields to have a good understanding on what is required for gatherings diagnostic logs.